src/devices/Ssd1306/Ssd1306.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Ssd1306.Command;
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ssd1306
+{
+    /// <summary>
+    /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+    /// light emitting diode dot-matrix graphic display system. 
+    /// </summary>
+    public class Ssd1306 : IDisposable
+    {
+        private I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Initializes new instance of Ssd1306 device that will communicate using I2C bus.
+        /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
+        /// light emitting diode dot-matrix graphic display system. 
+        /// </summary>
+        /// <param name="i2cDevice">>The I2C device used for communication.</param>
+        public Ssd1306(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+        }
+
+        /// <summary>
+        /// Send a command to the display controller.
+        /// </summary>
+        /// <param name="command">The command to send to the display controller.</param>
+        public void SendCommand(ICommand command)
+        {
+            byte[] commandBytes = command.GetBytes();
+            byte[] writeBuffer = new byte[commandBytes.Length + 1];
+            commandBytes.CopyTo(writeBuffer, 1);  // Control byte is 0x00 for first byte.
```

Comment:
We are still missing a null array check here.

Reply:
Also, why did we decide against using stackalloc?

Reply:
Updated